app/static/tests/unit/components/options/parameterSetView.test.ts

```diff
@@ -1,21 +1,38 @@
 import Vuex from "vuex";
 import { shallowMount } from "@vue/test-utils";
+import VueFeather from "vue-feather";
 import { BasicState } from "../../../../src/app/store/basic/state";
-import { mockRunState } from "../../../mocks";
+import { mockBasicState, mockRunState } from "../../../mocks";
 import ParameterSetView from "../../../../src/app/components/options/ParameterSetView.vue";
+import tooltip from "../../../../src/app/directives/tooltip";
+import { RunAction } from "../../../../src/app/store/run/actions";
 
 describe("ParameterSetView", () => {
-    it("renders as expected", () => {
+    const mockDeleteParameterSet = jest.fn();
+
+    beforeEach(() => {
+        jest.resetAllMocks();
+    });
+
+    const getWrapper = () => {
         const store = new Vuex.Store<BasicState>({
-            state: {
-                run: mockRunState({
-                    parameterValues: { alpha: 1, beta: 2, gamma: 3 }
-                })
-            } as any
+            state: mockBasicState(),
+            modules: {
+                run: {
+                    namespaced: true,
+                    state: mockRunState({
+                        parameterValues: { alpha: 1, beta: 2, gamma: 3 }
+                    }),
+                    actions: {
+                        [RunAction.DeleteParameterSet]: mockDeleteParameterSet
+                    }
+                }
+            }
         });
-        const wrapper = shallowMount(ParameterSetView, {
+        return shallowMount(ParameterSetView, {
             global: {
-                plugins: [store]
+                plugins: [store],
+                directives: { tooltip }
```

Comment:
I think this directive should be mocked.

Reply:
Good idea - updated as suggested.